bioconductor-singlecellexperiment-scripts-post-install-tests.sh

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env bash
+
+# This is a test script designed to test that everything works in the various
+# accessory scripts in this package. Parameters used have absolutely NO
+# relation to best practice and this should not be taken as a sensible
+# parameterisation for a workflow.
+
+function usage {
+    echo "usage: `basename "$0"` [action] [use_existing_outputs]"
+    echo "  - action: what action to take, 'test' or 'clean'"
+    echo "  - use_existing_outputs, 'true' or 'false'"
+    exit 1
+}
+
+action=${1:-'test'}
+use_existing_outputs=${2:-'false'}
+
+if [ "$action" != 'test' ] && [ "$action" != 'clean' ]; then
+    echo "Invalid action"
+    usage
+fi
+
+if [ "$use_existing_outputs" != 'true' ] && [ "$use_existing_outputs" != 'false' ]; then
+    echo "Invalid value ($use_existing_outputs) for 'use_existing_outputs'"
+    usage
+fi
+
+test_matrix_url="https://raw.githubusercontent.com/jdblischak/singleCellSeq/master/data/molecules.txt"
```

Comment:
Depending on how heavy these are, we might want to leave these files a space that we control? For instance the group google team space where we don't have a quota with the ebi user? There are easy ways to retrieve data from those places.